RELEASE_NOTES.md

```diff
@@ -1,5 +1,57 @@
 # Release Notes
 
+## 1.6.0
+### Features
+- The `podman network create`, `podman network rm`, `podman network inspect`, and `podman network ls` commands have been added to manage CNI networks used by Podman
+- The `podman volume create` command can now create and mount volumes with options, allowing volumes backed by NFS, tmpfs, and many other filesystems
+- Podman can now run containers without CGroups for better integration with systemd via the `--cgroups=disabled` flag to `podman create` and `podman run`. This is presently only supported with the `crun` OCI runtime
```

Comment:
suggest "systemd by using the `--cgroups=disabled` option with the `podman create` and `podman run` commands"